packages/swap/src/utils/fees.ts

```diff
@@ -121,3 +127,38 @@ export const calculateIncludedSwapFees = async (
     },
   ];
 };
+
+export const tryExtractFeesFromIngressAmount = async ({
```

Comment:
extracted the logic for calculating the "ingress" side fees (broker + ingress) here